### PR TITLE
[Documentation] Added a FAQ for executor returning successful even after the query fa…

### PR DESCRIPTION
**Pull request by sgururajshetty**
dded a FAQ for executor returning successful even after the query failed

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

File: docs/faq.md [hunk 1]
```diff
@@ -178,4 +179,9 @@ create datamap ag1 on table gdp21 using 'preaggregate' as select cntry, sum(gdp)
 select cntry,sum(gdp) from gdp21,pop1 where cntry=ctry group by cntry;
 ```
 
+## Why all executors are returning success even after some of the query failed?
```
Comment (sraghunandan) [inline docs/faq.md]: language is not correct. its not a query, it a command.We need to say why executor does not retry and why the signal sent to driver is not interpreted as failed

**Comment by sgururajshetty** [comment]
@sraghunandan kindly review

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://88.99.58.216:8080/job/ApacheCarbonPRBuilder/3037/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/4281/

**Comment by sgururajshetty** [comment]
Closed as this FAQ is not needed

**Pull request closed without merging**